lib/galaxy/config/schemas/config_schema.yml

```diff
@@ -2931,45 +2940,55 @@ mapping:
           a real admin user account via API.
           You should probably not set this on a production server.
 
-      ga4gh_service_id:
+      organization_name:
         type: str
         required: false
         desc: |
-          Service ID for GA4GH services (exposed via the service-info endpoint for the Galaxy DRS API).
-          If unset, one will be generated using the URL the target API requests are made against.
+          The name of the organization that operates this Galaxy instance. Serves as the
+          default for the GA4GH service organization name and can be exposed through Galaxy
+          markdown for reports and such. For instance, "Not Evil Corporation".
+
+          For GA4GH APIs, this is exposed via the service-info endpoint for the Galaxy DRS API.
+          If unset, one will be generated using ga4gh_service_id (but only in the context
+          of GA4GH APIs).
 
           For more information on GA4GH service definitions - check out
           https://github.com/ga4gh-discovery/ga4gh-service-registry
           and https://editor.swagger.io/?url=https://raw.githubusercontent.com/ga4gh-discovery/ga4gh-service-registry/develop/service-registry.yaml
 
-          This value should likely reflect your service's URL. For instance for usegalaxy.org
-          this value should be org.usegalaxy. Particular Galaxy implementations will treat this
-          value as a prefix and append the service type to this ID. For instance for the DRS
-          service "id" (available via the DRS API) for the above configuration value would be
-          org.usegalaxy.drs.
-
-      ga4gh_service_organization_name:
+      organization_url:
         type: str
         required: false
         desc: |
-          Service name for host organization (exposed via the service-info endpoint for the Galaxy DRS API).
-          If unset, one will be generated using ga4gh_service_id.
+          The URL of the organization that operates this Galaxy instance. Serves as the
+          default for the GA4GH service organization name and can be exposed through Galaxy
+          markdown for reports and such. For instance, "notevilcorp.com".
+
+          For GA4GH APIs, this is exposed via the service-info endpoint for the Galaxy DRS API.
+          If unset, one will be generated using ga4gh_service_id (but only in the context
+          of GA4GH APIs).
```

Comment:
This doesn't look correct, the default for `organization_url` is `f"{components.scheme}://{components.netloc}"` in `lib/galaxy/webapps/galaxy/api/drs.py`

Reply:
Good catch - you're right. I'll drop that line of the docs.